en/docs/setup/running-the-product.md

```diff
@@ -127,158 +116,68 @@ the Command Prompt as described below.
         $args \> /dev/null 2\>&1 &**
         
         !!! tip
-            The only change is replacing `             sh            ` with
-            `             bash            ` . This is required only for Solaris.
-    4.  Start the product by following the above instructions.
-
-### Accessing the management console
+            Replace `sh` with `bash` in above occurences.
+    4.  Start the product.
 
-Once the server has started, you can access the Management Console by
-opening a Web browser and typing in the management console's URL. The
-URL is displayed towards the end of the server start script's console
-and log. For example:
+## Accessing the management console
 
-![](attachments/34611935/44179743.png) 
+Once the server has started, you can access the Management Console by navigating 
+to the URL of the management console . The URL is displayed towards the end of the 
+server start script's console and log. The URL should be in the following format.
 
-The URL should be in the following format:
 `         https://<Server Host>:9443/carbon        `
 
-You can use this URL to access the Management Console on this computer
-from any other computer connected to the Internet or LAN. When accessing
-the Management Console from the same server where it is installed, you
-can type "localhost" instead of the IP address:
-`                   https://localhost:9443/carbon                 ` .
-
-!!! tip
-    The Management Console URL can be changed by modifying the
-    value of the `         MgtHostName        ` in the
-    `         <IS_HOME>/repository/conf/carbon.xml        ` file.
-    
-    ``` xml
-    <MgtHostName>localhost</MgtHostName>
-    ```
+Use the above URL to access the Management Console via any computer connected to 
+the Internet or LAN. When accessing the Management Console from the same server 
+where it is installed, type "localhost" instead of the IP address. Use **admin** 
+as both the username and password to log in to the management console.
 
+Example: `                   https://localhost:9443/carbon                 ` .
 
-At the sign-in screen, sign in to the Management Console using **admin**
-as both the username and password. You can then use the Management
-Console to manage the product. The tabs and menu items in the navigation
-pane on the left may vary depending on the features you have installed.
+!!! tip
 
-To view information about a particular page, click the **Help** link in
-the top right corner of that page, or click the **Docs** link to open
-this documentation for full information on managing the product.
+    The Management Console URL can be changed by modifying the
```

Comment:
Let's remove this config from here and add a link pointing to https://is.docs.wso2.com/en/5.9.0/setup/changing-the-hostname/

Reply:
Fixed: acbfac3f53618a2244bd6e2ab9766653f4d1edb9